format/index.md

```diff
@@ -1095,6 +1095,10 @@ strategy might use query parameters such as `page[number]` and `page[size]`,
 an offset-based strategy might use `page[offset]` and `page[limit]`, while a
 cursor-based strategy might use `page[cursor]`.
 
+> Note: The above example URI shows unencoded `[` and `]` characters simply
```

Comment:
Thanks again for this!

Two small things: since we don't actually have a URI here (but query parameters), I think this should say "The above example query parameters use unencoded `[` and `]` characters simply for readability." Then, since this note is already in the base spec, it should instead end: "In practice, these characters must be percent-encoded, per the requirements in [RFC 3986](http://tools.ietf.org/html/rfc3986#section-3.4)."